src/engine/enginebuffer.h

```diff
@@ -77,6 +78,9 @@ class EngineBuffer : public EngineObject {
     enum KeylockEngine {
         SOUNDTOUCH,
         RUBBERBAND,
+#if (RUBBERBAND_API_MINOR_VERSION >= 2 && RUBBERBAND_API_MINOR_VERSION >= 7)
```

Comment:
I think we can remove this guard here

Reply:
nope, the Keylock Engine combobox is populated with those values.

Reply:
Yes, but defining the enums value in any case will work.